arelle/plugin/validate/ESEF/ESEF_Current/ValidateXbrlFinally.py

```diff
@@ -669,7 +669,7 @@ def checkFootnote(elt: ModelInlineFootnote | ModelResource, text: str) -> None:
                         modelXbrl.error("ESEF.2.2.7.improperApplicationOfEscapeAttribute",
                                           _("Facts with datatype 'dtr-types:textBlockItemType' MUST use the 'escape' attribute set to 'true'. Facts with any other datatype MUST use the 'escape' attribute set to 'false' - fact %(conceptName)s"),
                                           modelObject=f, conceptName=f.concept.qname)
-                    if f.effectiveValue == "0" and f.xValue != 0:
+                    if f.effectiveValue in ["0", "-0"] and f.xValue != 0:
```

Comment:
Should we be handling this in the `effectiveValue` getter (converting `-0` to `0`)?